src/GUI/MainForm.Designer.cs

```diff
@@ -970,7 +970,8 @@ private void InitializeComponent()
             this.toolZoomValue.AutoSize = false;
             this.toolZoomValue.Enabled = false;
             this.toolZoomValue.Name = "toolZoomValue";
-            this.toolZoomValue.Size = new System.Drawing.Size(36, 22);
+            this.toolZoomValue.Font = new System.Drawing.Font(System.Drawing.FontFamily.GenericMonospace, 8);
```

Comment:
I would check here (between line 972/973) whether or not nclass is running on mono. You can use `Mono.IsRunningOnMono`. Otherwise you simply set the values like that, it looks really weird under windows.

Reply:
The problem is that I don't have a Windows box to test it, so I'm coding blindly. Maybe the problem is the size (just 8, maybe it is too small), or the FontFamily (monospaced, in contrast to san serif).

Another problem I'm worried about is that whether the code, as you proposed it, with an if, would still be possible to be edited from the designer. Maybe after loading it the layout it would go nuts.

Let me check out other possibilities.

Reply:
I would've not expected the designer to be that stupid, but you're correct, it flips out.

One possiblility is to revert your changes in MainForm.Designer.cs and add following in the constructor of MainForm after InitializeComponent (MainForm.cs, line ~47):

```csharp
if (MonoHelper.IsRunningOnMono)
{
    toolZoomValue.Font = new System.Drawing.Font(System.Drawing.FontFamily.GenericMonospace, 8);
    toolZoomValue.Size = new System.Drawing.Size(50, this.standardToolStrip.ClientSize.Height);
}
```

This will work without breaking the designer and under windows looks fine. I've tested it with inverted condition (as if I were under linux/mono) and the font shows fine.